pincer/middleware/presence_update.py

```diff
@@ -26,7 +26,9 @@ async def presence_update_middleware(self, payload: GatewayDispatch):
     """  # noqa: E501
     return (
         "on_presence_update",
-        PresenceUpdateEvent.from_dict(construct_client_dict(self, payload.data))
+        PresenceUpdateEvent.from_dict(
+            construct_client_dict(self, payload.data)
+        ),
```

Comment:
Quite pointless imo considering we bumped the length limit to 88 but I guess it can stay like this in fact

Reply:
I will revert this